libr/util/sys.c

```diff
@@ -460,6 +460,37 @@ R_API int r_sys_chdir(const char *s) {
 	return r_sandbox_chdir (s)==0;
 }
 
+R_API bool r_sys_aslr(int val) {
+	bool ret = true;
+#if __linux__
+	static const char *rva = "/proc/sys/kernel/randomize_va_space";
+	char buf[2] = {0};
+	snprintf(buf, sizeof (buf), "%d\n", val != 0 ? 2 : 0);
+	int fd = open (rva, O_WRONLY);
```

Comment:
use r_sandbox apis otherwise you will be bypassing the sandbox

Reply:
Yes good point.